src/drivers/device/vdev_posix.cpp

```diff
@@ -270,7 +270,7 @@ int px4_fsync(int fd)
 
 int px4_access(const char *pathname, int mode)
 {
-	if (mode == F_OK) {
```

Comment:
@mcharleb This cannot work for the orb_exists() function because we are calling it with F_OK.